app/docker/views/configs/edit/config.html

```diff
@@ -40,8 +40,8 @@
               <td>
                 <table class="table table-bordered table-condensed">
                   <tr ng-repeat="(k, v) in config.Labels">
-                    <td>{{ k }}</td>
-                    <td>{{ v }}</td>
+                    <td style="width:auto;word-break:break-word;word-break:break-all;white-space:pre-wrap;">{{ k }}</td>
+                    <td style="width:auto;word-break:break-word;word-break:break-all;white-space:pre-wrap;">{{ v }}</td>
```

Comment:
Could you please add it as a separate class in [app.css](https://github.com/portainer/portainer/blob/develop/assets/css/app.css)? It will make it way easier to maintain and use in other parts of the application!